src/main/java/de/muenchen/dave/domain/elasticsearch/detektor/Messstelle.java

```diff
@@ -0,0 +1,60 @@
+package de.muenchen.dave.domain.elasticsearch.detektor;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Data;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.elasticsearch.annotations.Document;
+import org.springframework.data.elasticsearch.annotations.Field;
+import org.springframework.data.elasticsearch.annotations.FieldType;
+import org.springframework.data.elasticsearch.annotations.GeoPointField;
+import org.springframework.data.elasticsearch.core.geo.GeoPoint;
+
+@Data
+@Document(indexName = "messstelle")
+public class Messstelle {
+
+    @Id
+    String id;
+
+    String nummer;
```

Comment:
Kann man das auch mstId nennen?

Reply:
Klar. Ich kann das auch umbenennen. Hab mich da nur am Schema der Zählstelle orientiert. Da hatten wir das auch **nummer** genannt

Reply:
Wenn möglich, sollten wir uns am neuen Datenmodell orientieren. Es sei denn, die Attribute müssen aus einem bestimmten Grund genau so heißen wie die in der Zählstelle. Aber ich glaube, das ist hier nicht der Fall.